.yamllint

```diff
@@ -0,0 +1,21 @@
+extends: default
+ignore: |
+  .tox
```

Comment:
Something like this probably:
```suggestion
ignore: |
  changelogs/changelog.yaml
  changelogs/fragments/
```
(We don't use tox, so no need to ignore `.tox`.)